canonical-web-and-design/ubuntu.com
# 7654 update openstack release table

## Done

Updated the OpenStack release cycle chart to match the spreadsheet

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/about/release-cycle#ubuntu-openstack-release-cycle
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the OpenStack release cycle chart data matches [the google sheet](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=1483484944)
- Check that the table in small screen view is also up to date


## Issue / Card

Fixes #7654
